.github/workflows/main.yml

```diff
@@ -167,6 +207,24 @@ jobs:
           --health-timeout 5s
           --health-retries 5
 
+      postgresql:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: db_user
+          POSTGRES_PASSWORD: db_password
+          POSTGRES_DB: db_name
+          # JIT is enabled by default since Postgres 17 and has a huge negative impact on performance,
```

Comment:
```suggestion
          # JIT is enabled by default since Postgres 17 and has a huge negative impact on tests performance,
```

Reply:
It affects not only test performance. See https://support.getgrist.com/self-managed/#what-is-a-home-database